sqlite-android/build.gradle

```diff
@@ -44,10 +41,10 @@ android {
 }
 
 dependencies {
-    implementation 'android.arch.persistence:db:1.0.0'
-    implementation 'com.android.support:support-core-utils:26.1.0'
-    androidTestImplementation 'com.android.support.test:runner:0.5'
-    androidTestImplementation 'com.android.support.test:rules:0.5'
+    implementation 'android.arch.persistence:db:1.1.1'
+    implementation 'com.android.support:support-core-utils:28.0.0'
+    androidTestImplementation 'com.android.support.test:runner:1.0.2'
+    androidTestImplementation 'com.android.support.test:rules:1.0.2'
 }
```

Comment:
The test stuff is the biggest - if you're not familiar with it, basically it's all annotations now, you don't extend or implement anything. Then you just tag things carefully and you should be migrated. With the exception of my poor understanding of how performance test suites work - so I may have blown that up.